mirrord-protocol/src/tcp.rs

```diff
@@ -37,4 +37,6 @@ pub enum DaemonTcp {
     NewConnection(NewTcpConnection),
     Data(TcpData),
     Close(TcpClose),
+    /// Used to notify the subscription occured
```

Comment:
Since this doesn't serve any actual function in the product, I would also document why it exists